### PR TITLE
Changelog entry for named arguments

### PR DESCRIPTION
**Pull request by Bolpat**
Named arguments are implemented in #14575 and #14776.

### CODE REVIEW

File: changelog/dmd.named-arguments.dd [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Named arguments have been implemented
+
+In a function call expression, an argument can new be prefixed by the identifier of the parameter it binds to.
```
Comment (RazvanN7) [inline changelog/dmd.named-arguments.dd:3]: Could you add a link to the DIP? That way, interested users will have more information.

@dkorpel I think we should also create a spec PR.

Comment (dkorpel) [reply changelog/dmd.named-arguments.dd:3]: I was going to add a spec PR and changelog (https://github.com/dkorpel/dmd/commit/8f72ce878127a2148634be666537066a90208e6b), but the implementation is unfinished (doesn't work with IFTI yet) so I planned to fix that first.

File: changelog/dmd.named-arguments.dd [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    assert(f(s, i, t) == "B");
+    // f(s, t); // error, neither A nor B match
+    // f(t, s); // error, neither A nor B match
+
+    // named arguments
+    // f(s: s, t: t, i: i); // error, ambiguous
+    assert(f(s: s, t: t) == "B");
+    assert(f(t: t, s: s) == "B");
+    assert(f(t: t, i, s: s) == "A");
+    assert(f(s: s, t: t, i) == "A");
```
Comment (schveiguy) [inline changelog/dmd.named-arguments.dd:28]: The single letter parameter names and argument names make this really hard to follow. I'd suggest, since there are no "refs" here, just use literals. And change the parameter names to something like `tVal`, `intVal`, etc.

The changelog is to help understand the feature. This kind of brevity/abstraction can be fine in a spec document.

**Comment by Bolpat** [comment]
I’ll close this as the changelog was finished on 1st of April. If that is actually not the case, please tell me (or just reopen).
